driver/sccb.c

```diff
@@ -41,7 +41,7 @@ int SCCB_Init(int pin_sda, int pin_scl)
 {
     ESP_LOGI(TAG, "pin_sda %d pin_scl %d\n", pin_sda, pin_scl);
     //log_i("SCCB_Init start");
-    i2c_config_t conf;
+    i2c_config_t conf = {};
```

Comment:
I would suggest to leave the line as-is and then add next line to be:
```c
memset(&conf, 0, sizeof(i2c_config_t));
```

Reply:
changed it as suggested with d70d104